components/cdc/src/observer.rs

```diff
@@ -106,6 +181,11 @@ impl<E: KvEngine> CmdObserver<E> for CdcObserver {
     ) {
         assert!(!cmd_batches.is_empty());
         fail_point!("before_cdc_flush_apply");
+        // Untrack raw ts regardless of the ob level
+        // Because RawTrackTask cannot get the ob level in
```

Comment:
Suggest: Because RawKV locks is tracked regradless of observe level, as it is in Raft propose procedure and can not get an accurate observe level.